lib/xlsx/xform/sheet/worksheet-xform.js

```diff
@@ -368,9 +369,11 @@ class WorkSheetXform extends BaseXform {
       return true;
     }
 
-    this.parser = this.map[node.name];
-    if (this.parser) {
-      this.parser.parseOpen(node);
+    if (!this.ignoreNodes.includes(node.name)) {
+      this.parser = this.map[node.name];
+      if (this.parser) {
+        this.parser.parseOpen(node);
+      }
```

Comment:
```suggestion
    if (this.map[node.name] && !this.ignoreNodes.includes(node.name)) {
      this.parser = this.map[node.name];
      this.parser.parseOpen(node);
```

I would like to propose that on purpose to reduce nested ifs